data/json/items/ammo.json

```
@@ -467,7 +467,7 @@
     "id": "bb",
     "price": 500,
     "price_postapoc": 50,
-    "name": { "str": "BB" },
+    "name": { "str_sp": "BBs" },
```

```suggestion
    "name": { "str": "BB" },
    "name": { "str_pl": "BBs" },
```

Remember, str_sp is used for both singular *and* plural; "1 BBs" is just as wrong as "50 BB."

---

Then "box of BBs" vs "boxes of BBs"?

---

Oh, wait, yes, apparently. Its description is a box of BBs, but ... is this a single BB in game? I can't tell.

---

The "count" field says it's 500 units of ammunition, as far as I can tell.

---

With the below correction from Qrox, I think just putting it back to "str": "BB" is preferable.